dbhydra/dbhydra_core.py

```diff
@@ -1669,24 +1726,26 @@ def modify_column(self, column_name, new_column_type):
 
 
 class XlsxDB(AbstractDB):
-    def __init__(self, config_file="config.ini", db_details=None):
+    def __init__(self, config_file="config.ini", db_details=None, db_directory_path: Path = Path("database")):
```

Comment:
wrong - it cannot have different inputs than AbstractDB - would lose compatibility with other Dialects

Reply:
db_directory_path should be added in db_details rather